templates/themes/horizon/invenio_communities/details/home/index.html

```diff
@@ -119,19 +119,18 @@ <h1 class="ui large header">{{ _("Recent uploads") }}</h1>
       </div>
     {% endif %}
 
-    <div class="ui stackable theme-font grid container rel-mt-2">
+    <div class="ui stackable theme-font one column grid container rel-mt-2">
       <div class="column">
           <div class="row item">
             <h1 class="ui large header">{{ _("How it works") }}</h1>
           </div>
-          <div class="ui divider  ml-0"></div>
-          <div class="sixteen wide stretched column rel-mt-4">
-            <div class="ui equal width grid">
-              <div class="five wide computer sixteen wide mobile  segment-container column rel-m-1">
-                <section>
+          <div class="ui divider ml-0"></div>
+          <div class="ui wide stretched stackable three column grid rel-mt-3">
```

Comment:
Before we had a `sixteen` grid, with 3 children having a width of `five` each (total of 15).
The added `rel-m-1` on each column was adding extra width to each column making only 15 and not 16 fit, but also breaking responsivness with 2 blocks on one row and 1 block on a next row.

Now we use a grid with `three` columns, and we do not add extra margin to the direct `column` elements to not break Semantic UI's grid layout.